pybamm/solvers/casadi_solver.py

```diff
@@ -186,12 +187,9 @@ def _integrate(self, model, t_eval, inputs=None):
                     count += 1
                     if count >= self.max_step_decrease_count:
                         raise pybamm.SolverError(
-                            """
-                            Maximum number of decreased steps occurred at t={}. Try
-                            solving the model up to this time only or reducing dt_max.
-                            """.format(
-                                t
-                            )
+                            "Maximum number of decreased steps occurred at t={}. Try "
+                            "solving the model up to this time only or reducing dt_max."
+                            "".format(t)
```

Comment:
should we give the dimensional time in the error to make it easier for people to pass to `t_eval`. similar comment for `dt_max`